### PR TITLE
Feature/contacts page

### PR DESCRIPTION
**Pull request by olgatim**

### CODE REVIEW

File: src/scenes/ContactsPage/components/Description/Description.module.scss [hunk 1]
```diff
@@ -1,126 +1,140 @@
...
+  @include hl-margin(bottom, 100px);
   text-align: center;
+  letter-spacing: -2px;
+  background: linear-gradient(
+    125.56deg,
+    #e7a28f 3.52%,
+    #f9d6ac 41.89%,
+    #fbfefc 96.77%
+  );
+  -webkit-background-clip: text;
```
Comment (HLOkeksandr) [inline src/scenes/ContactsPage/components/Description/Description.module.scss]: `-webkit-background-clip: text;`
I think you can remove this.

File: src/scenes/ContactsPage/components/Description/Description.module.scss [hunk 2]
```diff
@@ -1,126 +1,140 @@
...
   text-decoration: none;
   cursor: pointer;
   white-space: nowrap;
+  background: linear-gradient(
+    130.51deg,
+    #e7a28f 3.52%,
+    #f9d6ac 41.89%,
+    #fbfefc 96.77%
+  );
+  -webkit-background-clip: text;
```
Comment (HLOkeksandr) [inline src/scenes/ContactsPage/components/Description/Description.module.scss]: `-webkit-background-clip: text;`
I think you can remove this.

**Comment by lazio** [comment]
approved with a few comments

File: src/components/GradientText/GradientText.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './GradientText.module.scss';
+
+const gradientText = ({ text, background }) => {
```
Comment (lazio) [inline src/components/GradientText/GradientText.js]: minor, but the component's name should start with uppercase letter `gradientText` -> `GradientText `

**Pull request merged**